lib/unifex/code_generator/base_types/int.ex

```diff
@@ -12,9 +12,9 @@ defmodule Unifex.CodeGenerator.BaseTypes.Int do
     def generate_arg_parse(argument, name, _ctx) do
       ~g"""
       ({
-        long long #{name}_longlong;
-        int result = ei_decode_longlong(#{argument}->buff, #{argument}->index, &#{name}_longlong);
-        #{name} = (int)#{name}_longlong;
+        long long tmp_longlong;
+        int result = ei_decode_longlong(#{argument}->buff, #{argument}->index, &tmp_longlong);
+        #{name} = (int)tmp_longlong;
```

Comment:
This was for compatibility with lists. In case of parsing lists we can end up with `in_list[i] _longlong`